zeroX:  200.0
zeroY:  0.0
zeroZ:  -30
zeroR:  0.0
